cves/kernel/CVE-2019-15807.yml

```diff
@@ -133,10 +127,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: "No unit tests were found in the surrounding project."
+  fix: true
```

Comment:
Doesn't this mean the fix answer should be false? Looking at the fix commit, it is a 2 line code change with no updates to any unit tests. It looks like only production code is updated, can you clarify a little more?